lib/checks/aria/aria-roledescription.js

```diff
@@ -0,0 +1,20 @@
+const role = axe.commons.aria.getRole(node);
+const supportedRoles = [
```

Comment:
Should we not make these configurable options? This way it can be changed in custom rules to

1. Support different AT environments (e.g. internal employee application with a controlled AT environment), and
1. Be changed easily in custom rules if support changes without having to wait for an axe-core release

Reply:
Good point